azext_edge/edge/util/common.py

```diff
@@ -19,6 +19,20 @@
 logger = get_logger(__name__)
 
 
+def parse_kvp_nargs(kvp_nargs: List[str]) -> dict:
+    """
+    Parses kvp nargs into a dict handling values of null and empty string.
+    """
+    result = {}
+    if not kvp_nargs:
+        return result
+
+    for item in kvp_nargs:
+        key, sep, value = item.partition("=")
+        result[key] = value if sep else None
```

Comment:
This accepts inputs like "potato" -> {"potato": None}

I thought the og command (below) did not take these cases for a reason? Is that reason still valid

(PS: I do like the simplicity of this more - just double checking)

Reply:
Correct. This function would replace most usage of the prior function. The null case is useful and necessary, for example it is used to delete a PATCH configuration setting. It is better in every way. Look at the unit test for all the cases.